turbine/src/cluster_nodes.rs

```diff
@@ -395,7 +401,7 @@ fn cmp_nodes_stake(a: &Node, b: &Node) -> Ordering {
 // same TVU socket-addr, we only send shreds to one of them.
 // Additionally limits number of nodes at the same IP address to
 // MAX_NUM_NODES_PER_IP_ADDRESS.
-fn dedup_tvu_addrs(nodes: &mut Vec<Node>) {
+fn dedup_tvu_addrs(nodes: &mut Vec<Node>, keep_identity: Pubkey) {
```

Comment:
I think we can either just call this self_pubkey or fully commit to the bit that this could generically be any pubkey that we want to force keep (and thus update the comment on line 418 below)

Reply:
addressed in [92202a0](https://github.com/anza-xyz/agave/pull/5380/commits/92202a07babf5e31fe1d1bc5cb585a539d770414)